src/LiveComponent/assets/dist/live_controller.js

```diff
@@ -1375,14 +1375,14 @@ function executeMorphdom(rootFromElement, rootToElement, modifiedFieldElements,
                     return false;
                 }
                 if (fromEl instanceof HTMLElement && toEl instanceof HTMLElement) {
-                    if (typeof fromEl.__x !== 'undefined') {
+                    if (typeof fromEl.__x !== 'undefined' || typeof fromEl._x_dataStack !== 'undefined') {
                         if (!window.Alpine) {
                             throw new Error('Unable to access Alpine.js though the global window.Alpine variable. Please make sure Alpine.js is loaded before Symfony UX LiveComponent.');
                         }
                         if (typeof window.Alpine.morph !== 'function') {
                             throw new Error('Unable to access Alpine.js morph function. Please make sure the Alpine.js Morph plugin is installed and loaded, see https://alpinejs.dev/plugins/morph for more information.');
                         }
-                        window.Alpine.morph(fromEl.__x, toEl);
+                        window.Alpine.morph(fromEl.__x || fromEl, toEl);
```

Comment:
Passing fromEl here and not fromEl.__x seems weird to me, as depending on what the window.Alpine.morph will do, the following of the code can be deeply impacted here :| 

cloneHTMLElement(fromEl) for instance would return wrong results if some morphing is done on the element before

Or am I missing something ?